code/mission/missionparse.cpp

```diff
@@ -2445,6 +2445,15 @@ int parse_create_object_sub(p_object *p_objp)
 		}
 	}
 
+	// if this is an asteroid target, add it to the list
+	if (!Asteroid_target_ships.empty()) {
+		for (SCP_string& name : Asteroid_target_ships) {
+			if (strcmp(name.c_str(), shipp->ship_name) == 0) {
```

Comment:
I would use `stricmp` since that is typically used elsewhere in parsing situations.